alacritty/src/renderer/mod.rs

```diff
@@ -30,21 +30,19 @@ use crate::config::ui_config::{Delta, UIConfig};
 use crate::cursor;
 use crate::gl;
 use crate::gl::types::*;
+use crate::renderer::rect::{RectRenderer, RectShaderProgram};
 use crate::renderer::rects::RenderRect;
```

Comment:
`rects` feel like a publicly-visible module that is itself "logic only", without any rendering.
I can (1) rename `rect.rs` to `rect_render.rs` or something. Or I can just (2) dump renderer into `rects.rs`.
Or is there a better option?

Reply:
You can always rearrange things to make more sense logically. We just shouldn't have two modules with nearly identical names. Even `rect.rs` and `rect_render.rs` are so close together that it raises the question why those are not combined.

There's also nothing wrong with having part of a module for public use. But if the two things don't make sense together, it's possible to restructure things. Some stuff might be better in a `display/` module maybe. But we can also delay that until the rest of this PR is figured out. Since this is mostly an organizational issue.